code/modules/research/designs/weapon_designs.dm

```diff
@@ -221,24 +231,24 @@
 	category = list("Ammo")
 
 /datum/design/mag_smg/ap_mag
-	name = "Submachine Gun Armour Piercing Magazine (9mmAP)"
-	desc = "A 42-round armour piercing magazine for the prototype submachine gun. Deals slightly less damage by bypasses most armour"
+	name = "SABR SMG Armour Piercing Magazine (9mmAP)"
+	desc = "A 30-round armour piercing magazine for the prototype submachine gun. Deals slightly less damage by bypasses most armour"
 	id = "mag_smg_ap"
-	materials = list(MAT_METAL = 3000, MAG_SILVER = 100)
+	materials = list(MAT_METAL = 3000, MAT_SILVER = 100)
 	build_path = /obj/item/ammo_box/magazine/smgm9mm/ap
 
 /datum/design/mag_smg/incin_mag
-	name = "Submachine Gun Incindiary Magazine (9mmIC)"
-	desc = "A 42-round incindiary round magazine for the prototype submachine gun. Deals significanlty less damage but sets the target on fire"
+	name = "SABR SMG Incendiary Magazine (9mmIC)"
+	desc = "A 30-round incindiary round magazine for the prototype submachine gun. Deals significanlty less damage but sets the target on fire"
 	id = "mag_smg_ic"
-	materials = list(MAT_METAL = 3000, MAG_SILVER = 100, MAT_GLASS = 400)
+	materials = list(MAT_METAL = 3000, MAT_SILVER = 100, MAT_GLASS = 400)
 	build_path = /obj/item/ammo_box/magazine/smgm9mm/fire
 
 /datum/design/mag_smg/incin_tox
-	name = "Submachine Gun Urnaium Magazine (9mmIC)"
-	desc = "A 42-round urnaium tipped round magazine for the prototype submachine gun. Deals toxin damage, but less overall damage."
+	name = "SABR SMG Urnaium Magazine (9mmTX)"
+	desc = "A 30-round urnaium tipped round magazine for the prototype submachine gun. Deals toxin damage, but less overall damage."
 	id = "mag_smg_tx"
-	materials = list(MAT_METAL = 3000, MAT_GLASS = 200, MAG_URNAIUM = 1000)
+	materials = list(MAT_METAL = 3000, MAT_GLASS = 200, MAT_URNAIUM = 1000)
```

Comment:
Do not forget to change it to "URANIUM". Please remember to test and verify each portion of your PR in order to catch such bugs.